ggshield/output/json/json_output.py

```diff
@@ -106,7 +106,7 @@ def flattened_policy_break(
             "total_occurrences": len(policy_breaks),
         }
         for policy_break in policy_breaks:
-            update_policy_break_matches(policy_break.matches, lines, is_patch)
+            update_policy_break_matches(policy_break.matches, lines, is_patch, True)
```

Comment:
I don't get why you did not need to set this parameter to True for non json output

Reply:
ok, seen with you